packs/github/sensors/github_repository_sensor.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, sensor_service, config=None, poll_interval=None):
         self._last_event_ids = {}
 
     def setup(self):
-        self._client = Github(self._config['token'])
+        self._client = Github(self._config['token'] or None)
```

Comment:
`self._config.get('token', None)` will break it again.

when `token = ""` (empty string).